services/streammanager/lambda/lambdamanager.go

```diff
@@ -0,0 +1,92 @@
+//go:generate mockgen --build_flags=--mod=mod -destination=../../../mocks/services/streammanager/lambda/mock_lambda.go -package mock_lambda github.com/rudderlabs/rudder-server/services/streammanager/lambda LambdaClient
+
+package lambda
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/mitchellh/mapstructure"
+	"github.com/rudderlabs/rudder-server/services/streammanager/common"
+	"github.com/rudderlabs/rudder-server/utils/awsutils"
+	"github.com/rudderlabs/rudder-server/utils/logger"
+	"github.com/tidwall/gjson"
+)
+
+// Config is the config that is required to send data to Lambda
+type Config struct {
```

Comment:
this internal structure doesn't need to be exported

Reply:
updated